vector/src/main/java/im/vector/app/AutoRageShaker.kt

```diff
@@ -221,17 +224,19 @@ class AutoRageShaker @Inject constructor(
                 withCrashLogs = true,
                 withKeyRequestHistory = true,
                 withScreenshot = false,
-                theBugDescription = "UISI detected $matchingIssue",
+                theBugDescription = "Auto-reporting decryption error \nRecipient rageshake: $matchingIssue",
```

Comment:
If we want this to be totally consistent with element-web, it'd be
```suggestion
                theBugDescription = "Auto-reporting decryption error (sender)\nRecipient rageshake: $matchingIssue",
```
but that's not a huge deal, and we have the labels now in any case so maybe element-web doesn't need this either anymore.

Reply:
updated